lib/instrumental/capistrano/capistrano2.rb

```diff
@@ -34,7 +34,7 @@
       deploy_duration_in_seconds = end_at - start_at
       deployer                   = fetch(:deployer)
       agent_options              = { :synchronous => true }
-      agent_options[:collector]  = instrumental_host if fetch(:instrumental_host)
+      agent_options[:collector]  = fetch(:instrumental_host, false)
```

Comment:
Sorry, needs to be `instrumental_host if fetch(:instrumental_host, false)`. I could have been more clear in my other message.

Reply:
Ah, wasn't sure. Easy enough tho.